Scripts/script-CommonServer.yml

```diff
@@ -230,6 +230,169 @@ script: |-
       var ret_string = '_'+string;
       return ret_string.replace(/_([a-z])/g, function (g) { return g[1].toUpperCase(); });
   };
+  
+  /**
+   * Converts dots into spaces and capitalizes
+  */
+  var dotToSpace = function(string){
+      var ret_string = '.'+string;
+      return ret_string.replace(/\.([a-z,A-Z])/g, function (g) { return ' '+g[1].toUpperCase(); });
+  };
+  
+  function mapObjFunction(mapFields, filter) {
+      var transformSingleObj= function(obj) {
+          var res = {};
+          mapFields.forEach(function(f) {
+              if(!filter || filter(f)){
+                  res[f.to] = dq(obj, f.from);
+              }
+          });
+          return res;
+      };
+      return function(obj) {
+          if (obj instanceof Array) {
+              var res = [];
+              for (var j=0; j < obj.length; j++) {
+                  var current = transformSingleObj(obj[j]);
+                  if (current) {
+                      res.push(current);
+                  }
+              }
+              return res;
+          }
+          return transformSingleObj(obj);
+      };
+  }
+
+  function createContext(data, id) {
+      var createContextSingle = function(obj) {
+          var res = {};
+          var keys = Object.keys(obj);
+          keys.forEach(function(k) {
+              var values = k.split('.');
+              var current = res;
+              for (var j = 0; j<values.length - 1; j++) {
+                  if (!current[values[j]]) {
+                      current[values[j]] = {};
+                  }
+                  current = current[values[j]];
+              }
+              current[values[j]] = obj[k];
+          });
+          if (!res.ID && id) {
+              res.ID = id;
+          }
+          return res;
+      };
+      if (data instanceof Array) {
+          var res = [];
+          for (var j=0; j < data.length; j++) {
+              res.push(createContextSingle(data[j]));
+          }
+          return res;
+      }
+      return createContextSingle(data);
+  }
+
+  function isFunction(functionToCheck) {
+   var getType = {};
+   return functionToCheck && getType.toString.call(functionToCheck) === '[object Function]';
```

Comment:
This looks a bit ugly and also not indented... I think you can:
```js
return functionToCheck && typeof functionToCheck === 'function';
```

Reply:
@slavikm changed